include/wx/rawbmp.h

```diff
@@ -722,7 +740,11 @@ struct wxPixelDataOut<wxBitmap>
                 operator bool() const
                 {
                     wxByte mask = static_cast<wxByte>(1 << m_bit);
+                #ifndef __WXQT__
```

Comment:
This looks like the indicator that a mask in wxQt is reversed compared to wx conventions which is surely problematic and it looks like it would be better to fix it by inverting the mask when constructing it?

Reply:
It's unfortunate that we have to reverse the logic here, because the code checking to see whether the current pixel is transparent or not uses this `bool operator` to determine that. and **Qt** and **wx** seem to operate in reverse manner when applying the mask  to bitmap.